contrib/docker/Dockerfile

```diff
@@ -3,13 +3,13 @@
 ###############
 
 # Basic Go environment with git, SSL CA certs, and upx.
-# The image below is golang:1.21.5-alpine3.18 (linux/amd64)
+# The image below is golang:1.21.6-alpine3.19 (linux/amd64)
 # It's pulled by the digest (immutable id) to avoid supply-chain attacks.
 # Maintainer Note:
 #    To update to a new digest, you must first manually pull the new image:
 #    `docker pull golang:<new version>`
 #    Docker will print the digest of the new image after the pull has finished.
-FROM golang@sha256:5c1cabd9a3c6851a3e18735a2c133fbd8f67fe37eb3203318b7af2ffd2547095 AS builder
+FROM golang@sha256:771b7685c731a02d4643ab7b3103d95fc64eb091ee1751c662cbcdb0e65dcdcf AS builder
```

Comment:
I'm getting a different hash.

```
% docker pull golang:1.21.6-alpine3.19
1.21.6-alpine3.19: Pulling from library/golang
661ff4d9561e: Pull complete 
2979d3ffd3a4: Pull complete 
a32e8c70f577: Pull complete 
cedd8caa912b: Pull complete 
Digest: sha256:fd78f2fb1e49bcf343079bbbb851c936a18fc694df993cbddaa24ace0cc724c5
Status: Downloaded newer image for golang:1.21.6-alpine3.19
docker.io/library/golang:1.21.6-alpine3.19

```

Reply:
Hmm, it seems like every time I pull it right away it ends up changing shortly after.  I pulled it again and now it matches what you posted too.  I guess I need to wait a bit in the future.  Not sure what's up there.  Maybe something to do with it changing when the architectures are updated or something.

Updated.

Reply:
Thanks for double checking, by the way!

Reply:
6 hours later an I am seeing `sha256:fd78f2fb1e49bcf343079bbbb851c936a18fc694df993cbddaa24ace0cc724c5`.

Looks like it has stabilized.